docs/common/registration.md

```diff
@@ -33,7 +33,7 @@ To register as an OSG contact or add your GitHub ID to your pre-existing contact
 - **If you cannot find your name**, send an email to <mailto:help@opensciencegrid.org> with the following information:
     - Full name
     - Primary email address
-    - GitHub ID
+    - GitHub account name
```

Comment:
What's the difference?

Reply:
When I wrote "GitHub ID" I really meant username; sorry if that's unclear.

Reply:
The ID is a bunch of numbers

Reply:
Can we change it to "GitHub user name" instead? "account name" sounds kinda weird -- in general that's not the user-facing terminology.